docker-compose.yml

```diff
@@ -418,6 +420,178 @@ services:
       KAFKA_KAFKA_REST_CONFLUENT_METADATA_SERVER_URLS_MAX_AGE_MS: 60000
       KAFKA_KAFKA_REST_CLIENT_CONFLUENT_METADATA_SERVER_URLS_MAX_AGE_MS: 60000
         
+  kafka3:
```

Comment:
Hmmm, my comment got lost, I'll add again.  I think this PR should enable SBC with minimal impact to current demo, so I propose making the following changes:

- create a new folder `scripts/sbc`
- move add/kill script there
- use `certs-create-one-user.sh` instead of modifying `certs-create.sh`
- use a separate docker-compose file for kafka3 instead of modifying this base one

And it really brings into question whether the partition count should be modified in this PR or not, because that changes _does_ modify the base demo and everything needs to be re-validated end-to-end (code, docs, screenshots)

Reply:
Any room to negotiate on the don't modify `certs-create.sh` suggestion?

This one has the most potential to create problems in the workflow, since I assume this means that we would generate the cert for `kafka3` on-demand, or late, just prior to starting the `kafka3` broker as part of an SBC test-run.  It looks simpler to generate all the certs at the same time and build the keystores once.  Adding `kafka3` here amounts to creating a spare cert for the broker to use later ... if it's a significant concern I can look into how to safely extract this into its own lifecycle.

Reply:
The reasoning to create the `kafka3` cert later was to clearly articulate the boundary between the cp-demo base from the add-on SBC module, to not confuse folks who aren't exercising the SBC workflow, and to avoid unexpected problems down the road.  Since there is already a bash script to bring up the third broker, it would be a one line addition into that script to create the cert.  It's not a significant concern, NOT making the change won't block the PR, but I'm curious for the reasoning to not do this?

Reply:
Pushed a commit to separate `kafka3` certificate-generation.  Initial change for this had a bug which I've now fixed.